src/DrupalCheck.php

```diff
@@ -31,6 +32,11 @@ public function canRunInContext(ContextInterface $context): bool
   public static function getConfigurableOptions(): OptionsResolver
   {
       $resolver = new OptionsResolver();
+      $resolver->setDefaults([
+        'drupal_root' => null,
+      ]);
+      $resolver->addAllowedTypes('drupal_root', ['string']);
```

Comment:
This line fails when you don't have drupal_root set up in configuration.

```suggestion
      $resolver->addAllowedTypes('drupal_root', ['string', 'null']);
```